python/tvm/topi/cuda/conv2d_alter_op.py

```diff
@@ -345,4 +347,49 @@ def _conv2d_legalize(attrs, inputs, arg_types):
             else:
                 out = relay.nn.conv2d(data, kernel, **new_attrs)
             return out
+    elif data_dtype in ['float16', 'float32']:
```

Comment:
Why are we checking for 'float32'? My understanding is that tensorcores only work on smaller datatypes. You should consider looking at some of the support TVM has for running int8 and int4 workloads on tensorcores as well.

Reply:
#7146 added data type transform, so it's fine that both float16 and float32 can all be processed to support tensorcore.
But I'll still suggest @Meteorix to consider add some options in the whole flow that enables to close the tensor core feature manually, or add more strictly conditions here. In case if someone just don't want to use tensorcore or some shape/datatype cause special bugs.

Reply:
As discussed in https://github.com/apache/tvm/pull/7146 and https://github.com/apache/tvm/pull/7187, I'll make changes to support `float16` only and add some tests.